<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-CS6VK75P" failures="0" tests="38" name="com.ttt.TEST.TestNhanVien" time="0.139" errors="6" timestamp="2023-12-07T14:01:30 ICT" skipped="0">
  <testcase classname="com.ttt.TEST.TestNhanVien" name="checkTrungMaNV" time="0.033"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="XoaNhanVien_ThanhCong" time="0.001">
    <error message="
[public void com.ttt.TEST.TestNhanVien.XoaNhanVien_ThanhCong(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: XoaNhanVien_ThanhCong([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) NV2023072100002,(java.lang.String) Hồ Trọng Tường,(java.lang.String) Hậu Giang,(java.lang.String) tuong@gmail.com@gmail.com,(java.lang.String) 11111111,(java.lang.String) 1999-01-08,(java.lang.String) Thành công]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.ttt.TEST.TestNhanVien.XoaNhanVien_ThanhCong(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: XoaNhanVien_ThanhCong([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) NV2023072100002,(java.lang.String) Hồ Trọng Tường,(java.lang.String) Hậu Giang,(java.lang.String) tuong@gmail.com@gmail.com,(java.lang.String) 11111111,(java.lang.String) 1999-01-08,(java.lang.String) Thành công]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- XoaNhanVien_ThanhCong -->
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.005"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="deleteTest" time="0.000">
    <error message="
[public void com.ttt.TEST.TestNhanVien.deleteTest(java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: deleteTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) NV2023072100002,(java.lang.String) Hồ Trọng Tường,(java.lang.String) Hậu Giang,(java.lang.String) tuong@gmail.com@gmail.com,(java.lang.String) 11111111,(java.lang.String) 1999-01-08,(java.lang.String) Thành công]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.ttt.TEST.TestNhanVien.deleteTest(java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: deleteTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) NV2023072100002,(java.lang.String) Hồ Trọng Tường,(java.lang.String) Hậu Giang,(java.lang.String) tuong@gmail.com@gmail.com,(java.lang.String) 11111111,(java.lang.String) 1999-01-08,(java.lang.String) Thành công]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- deleteTest -->
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="checkNullTest" time="0.001">
    <error message="
[public void com.ttt.TEST.TestNhanVien.checkNullTest(java.lang.String,java.lang.String,int,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: checkNullTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=int, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) Nguyễn Thành Trung,(java.lang.String) trung@gmail.com,(java.lang.String) 1999-01-08,(java.lang.String) 11111111,(java.lang.String) 11111111,(java.lang.String) Thành công]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.ttt.TEST.TestNhanVien.checkNullTest(java.lang.String,java.lang.String,int,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: checkNullTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=int, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) Nguyễn Thành Trung,(java.lang.String) trung@gmail.com,(java.lang.String) 1999-01-08,(java.lang.String) 11111111,(java.lang.String) 11111111,(java.lang.String) Thành công]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1207)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- checkNullTest -->
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="XoaNhanVien_ThanhCong" time="0.001">
    <error message="
[public void com.ttt.TEST.TestNhanVien.XoaNhanVien_ThanhCong(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: XoaNhanVien_ThanhCong([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) NV2023072100002,(java.lang.String) Hồ Trọng Tường,(java.lang.String) Hậu Giang,(java.lang.String) tuong@gmail.com@gmail.com,(java.lang.String) 11111111,(java.lang.String) 1999-01-08,(java.lang.String) Thành công]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.ttt.TEST.TestNhanVien.XoaNhanVien_ThanhCong(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: XoaNhanVien_ThanhCong([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) NV2023072100002,(java.lang.String) Hồ Trọng Tường,(java.lang.String) Hậu Giang,(java.lang.String) tuong@gmail.com@gmail.com,(java.lang.String) 11111111,(java.lang.String) 1999-01-08,(java.lang.String) Thành công]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1207)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- XoaNhanVien_ThanhCong -->
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="checkTrungMaNV" time="0.039"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="checkNullTest" time="0.001">
    <error message="
[public void com.ttt.TEST.TestNhanVien.checkNullTest(java.lang.String,java.lang.String,int,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: checkNullTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=int, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) Nguyễn Thành Trung,(java.lang.String) trung@gmail.com,(java.lang.String) 1999-01-08,(java.lang.String) 11111111,(java.lang.String) 11111111,(java.lang.String) Thành công]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.ttt.TEST.TestNhanVien.checkNullTest(java.lang.String,java.lang.String,int,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: checkNullTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=int, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) Nguyễn Thành Trung,(java.lang.String) trung@gmail.com,(java.lang.String) 1999-01-08,(java.lang.String) 11111111,(java.lang.String) 11111111,(java.lang.String) Thành công]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- checkNullTest -->
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.001"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="checkTrungMaNV" time="0.025"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="deleteTest" time="0.000">
    <error message="
[public void com.ttt.TEST.TestNhanVien.deleteTest(java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: deleteTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) NV2023072100002,(java.lang.String) Hồ Trọng Tường,(java.lang.String) Hậu Giang,(java.lang.String) tuong@gmail.com@gmail.com,(java.lang.String) 11111111,(java.lang.String) 1999-01-08,(java.lang.String) Thành công]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.ttt.TEST.TestNhanVien.deleteTest(java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: deleteTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) NV2023072100002,(java.lang.String) Hồ Trọng Tường,(java.lang.String) Hậu Giang,(java.lang.String) tuong@gmail.com@gmail.com,(java.lang.String) 11111111,(java.lang.String) 1999-01-08,(java.lang.String) Thành công]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1207)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- deleteTest -->
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="UpdateNhanVien" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="checkTrungMaNV" time="0.023"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
  <testcase classname="com.ttt.TEST.TestNhanVien" name="themnvPass" time="0.000"/>
  <system-out/>
</testsuite> <!-- com.ttt.TEST.TestNhanVien -->
